contrib/kafka/pkg/apis/sources/v1alpha1/kafka_types.go

```diff
@@ -105,6 +120,9 @@ type KafkaSourceSpec struct {
 	// ServiceAccoutName is the name of the ServiceAccount that will be used to run the Receive
 	// Adapter Deployment.
 	ServiceAccountName string `json:"serviceAccountName,omitempty"`
+
+	// Resource limits and Request specifications of the Receive Adapter Deployment
+	Resources KafkaResourceSpec `json:"resources,omitempty"`
```

Comment:
Why not corev1.ResourceRequirements?

Reply:
Updated in 37df1a3

Reply:
I was going to leave the same feedback but saw this, that it was done, and then that the change was reverted. Was there trouble with using corev1.ResourceRequirements?

Reply:
@danp please see the conversation in https://github.com/knative/eventing-sources/pull/399#discussion_r288702244
Short answer, yes, it led to error messages only visible within the controller logs